src/Qowaiv.Validation.DataAnnotations/Attributes/DefinedOnlyAttribute.cs

```diff
@@ -0,0 +1,60 @@
+using System.Reflection;
+
+namespace Qowaiv.Validation.DataAnnotations;
+
+/// <summary>Validates if the decorated enum has a value that is a defined.</summary>
+/// <typeparam name="TEnum">
+/// The type of the enum.
+/// </typeparam>
+[AttributeUsage(AttributeTargets.Property | AttributeTargets.Field | AttributeTargets.Parameter, AllowMultiple = false)]
+public class DefinedOnlyAttribute<TEnum> : ValidationAttribute
+    where TEnum : struct, Enum
+{
+    /// <summary>Initializes a new instance of the <see cref="DefinedOnlyAttribute{TEnum}"/> class.</summary>
+    public DefinedOnlyAttribute()
+        : base(() => QowaivValidationMessages.AllowedValuesAttribute_ValidationError) => Do.Nothing();
+
+    /// <summary>If true, for flag enums, also combinations of defined single values are allowed, that are not defined themselves explicitly.</summary>
+    /// <remarks>
+    /// When enabled, the logic falls back on <see cref="Enum.IsDefined(Type, object)"/>, as
+    /// that is stricter than, our implementation.
```

Comment:
```suggestion
    /// <summary>If true, combinations of defined single values, that are not defined explicitly themselves, are allowed for flag enums.</summary>
    /// <remarks>
    /// When enabled, the logic falls back on <see cref="Enum.IsDefined(Type, object)"/>,
    /// because that is stricter than our implementation.
```